.github/workflows/pr-integration.yml

```diff
@@ -8,6 +8,10 @@ jobs:
   build:
 
     runs-on: ubuntu-latest
+    defaults:
```

Comment:
credo che andrebbe rinominata in "web-api-integration", possibilmente anche il "name" nel dovrebbe dire la stessa cosa

Reply:
Quando aggiungo quella per la web app

Reply:
sì